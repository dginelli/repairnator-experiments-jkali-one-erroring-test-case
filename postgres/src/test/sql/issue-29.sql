SELECT * FROM zdb_tally('so_posts', 'owner_display_name', '^.*', '', 10, 'count');