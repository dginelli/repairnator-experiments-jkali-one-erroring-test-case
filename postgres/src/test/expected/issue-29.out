SELECT * FROM zdb_tally('so_posts', 'owner_display_name', '^.*', '', 10, 'count');
   term    | count  
-----------+--------
           | 154132
 user1249  |    598
 user2567  |    513
 user8685  |    254
 user4051  |    153
 user8     |     84
 user29981 |     79
 user39685 |     68
 user23157 |     50
 sergio    |     48
 user29079 |     47
(11 rows)

